l10n_ch_import_pain000/res/lsv-dd-test.yml

```diff
@@ -0,0 +1,375 @@
+-
+  In order to test lsv and dd file generation, I will create a customer with
```

Comment:
Is this a valid test for your pain module or is it just a copy from the lsv module? Please either rename it or remove it if it's not adapted.